azure-pipelines.yaml

```diff
@@ -5,10 +5,24 @@ trigger:
 
 jobs:
   - job: Release
-    condition: and(eq(variables['Build.Reason'], 'Manual'), eq(variables['release'], 'true')))
+    condition: or(eq(variables['release'], 'true'), ne(variables['Build.SourceBranchName'], 'master'))
```

Comment:
So release happens if the release variable is set or this is on a branch other than master?